bots/tests-scan

```diff
@@ -390,9 +421,10 @@ def scan_for_pull_tasks(api, policy, opts, repo):
     if opts.human_readable:
         func = lambda x: tests_human(*x)
         results.sort(reverse=True, key=lambda x: str(x))
-    else:
```

Comment:
The `else:` was accidentally (?) removed. "human view" is for, well, interactive human test descriptions. These should *not* get queued. You can check this locally with `bots/tests-scan -vd`, it should show running and pending tests.

Reply:
there's a return statement before the `with...`, as I didn't want to open the queue when called with `-vd`.